docs/source/techniques/ISISPowder-SampleDetails-v1.rst

```diff
@@ -234,6 +234,19 @@ Example Input:
 
     sample_obj.set_material(number_density=0.123, ...)
 
+.. _crystal_density_sampleDetails_isis-powder-diffraction-ref:
+
+crystal_density
+^^^^^^^^^^^^^^^
+This parameter defines the full or crystallographic density of the property.
+If this is not set it will use the `number_density` in its place
```

Comment:
If crystal_density is set but number_density is not then cystal dencity will be used for the PDFFourierTransform and everything else will use the default number density from set sample.

Reply:
It was more the behaviour if neither of the densities are set that I was asking about. The documentation you've written says that if crystal density is blank then it gets the same value as number_density but what if that's also blank?

Reply:
In number dencity is says
```
When :ref:`chemical_formula_sampleDetails_isis-powder-diffraction-ref`
defines an element this can automatically be calculated by Mantid.

If :ref:`chemical_formula_sampleDetails_isis-powder-diffraction-ref`
is not an element the user must enter this value.
```

I think this makes it clear that if not set the number dencity is calculated automatically for elements.

Reply:
OK - the bit that wasn't clear to me was whether this fallback for the number density got copied across into crystal density in the scenario where the users doesn't supply either. But not important, I'll approve now so this isn't hanging around over the holidays. Thanks